src/config/rc-files.md

```diff
@@ -0,0 +1,61 @@
+# rc.conf, rc.local and rc.shutdown
+
+The files `/etc/rc.conf`, `/etc/rc.local` and `/etc/rc.shutdown` can be used to
+configure certain parts of your Void system.
+
+## rc.conf
+
+Sourced in runit stages 1 and 3. This file can be used to set the following
+variables:
+
+### KEYMAP
+
+Specifies which keymap to use for the Linux console. Available keymaps are
+listed in `/usr/share/kbd/keymaps`. For example:
+
+```
+export KEYMAP=fr
+```
+
+For further details, refer to
+[loadkeys(1)](https://man.voidlinux.org/loadkeys.1).
+
+### TIMEZONE
+
+Specifies which timezone to use. Available timezones are listed in
+`/usr/share/zoneinfo`. For example:
+
+```
+export TIMEZONE=Europe/Berlin
+```
+
+### HARDWARECLOCK
+
+Specifies whether the hardware clock is set to UTC or local time.
+
+By default this is set to `utc`. However, Windows sets the hardware clock to
+local time, so if you are dual-booting with Windows, you need to either
+configure Windows to use UTC, or set this variable to `localtime`.
+
+For further details, refer to [hwclock(8)](https://man.voidlinux.org/hwclock.8).
+
+### FONT
+
+Specifies which font to use for the Linux console. Available fonts are listed in
+`/usr/share/kbd/consolefonts`. For example:
+
+```
+export FONT=eurlatgr
+```
+
+For further details, refer to [setfont(1)](https://man.voidlinux.org/setfont.1).
+
+## rc.local
```

Comment:
Mention that these are assumed to be shell scripts, while rc.conf is normally just the variables to set?

Reply:
Added in https://github.com/void-linux/void-docs/commit/d010d217ab8788d2feffb6711fa2825f53fbe4c3.